trieste/models/gpflux/sampler.py

```diff
@@ -20,9 +20,21 @@
 from gpflux.sampling.sample import Sample
 
 from ...types import TensorType
+from ...utils import DEFAULTS
+from ..interfaces import ReparametrizationSampler, TrajectoryFunction, TrajectorySampler
+from .interface import GPfluxPredictor
 
 
 def sample_consistent_lv_layer(layer: LatentVariableLayer) -> Sample:
+    r"""
+    Returns a gpflux.sampling.sample.Sample object which allows for consistent sampling from a given
+    gpflux.layers.LatentVariableLayer.
+
+    :param layer: the gpflux.layers.LatentVariableLayer to obtain samples from
```

Comment:
type is clear from typing, so no need to state it again
```suggestion
    :param layer: The gpflux layer to obtain samples from.
```

Reply:
Sure, happy to change it, but it's not clear to me why we then need it in the overall description (as in the comment you left above)?